lib/crawler_web/controllers/user_session_controller.ex

```diff
@@ -0,0 +1,18 @@
+defmodule CrawlerWeb.UserSessionController do
+  use CrawlerWeb, :controller
+
+  alias Crawler.Account.Accounts
+  alias CrawlerWeb.UserAuth
+
+  def new(conn, _params), do: render(conn, "new.html", error_message: nil)
```

Comment:
I think we no need to set the `error_message: nil` here as it should be nil in most case

Reply:
@andyduong1920  It seemed we need the `error_message`, because it's used in the view file. Catch me if I'm wrong.

Reply:
What happens if we remove the `error_message: nil` here @vnntsu do you have any error on your side, a screenshot will be better

Reply:
@andyduong1920 Here we are, we show the error when log in failed

![Screenshot 2022-12-14 at 10 26 20](https://user-images.githubusercontent.com/17830319/207497960-b4782a94-b448-4d07-bea6-47b42db33f8c.png)
![Screenshot_2022-12-14_at_10_26_58](https://user-images.githubusercontent.com/17830319/207498065-6de8c6f0-a346-40e7-bcdf-6704fbf95cea.png)

Reply:
@vnntsu I think we miss-understanding, my question is `what happens when we remove the error_message: nil` not a question the `error_message: gettext` behaviour :)

Reply:
oh, got you, we retrieve the value of `error_message` at HTML, so if we don't pass `nil`, it's gonna be an error.
![Screenshot 2022-12-14 at 10 51 14](https://user-images.githubusercontent.com/17830319/207501455-c9a71a49-6526-47e5-a056-2c1f9a57b1f9.png)
![Screenshot 2022-12-14 at 10 53 29](https://user-images.githubusercontent.com/17830319/207501549-8baf8791-d78b-48d4-bbeb-51d9a1c1d30b.png)

Reply:
ah I see, normally we don't generate the error like this, maybe you can have a look on the [Phoenix Flash](https://hexdocs.pm/phoenix/controllers.html#flash-messages)

Reply:
@andyduong1920 updated in [ace81f9](https://github.com/vnntsu/elixir-ic/pull/33/commits/ace81f905b09e6eedc84241723f8319974bab9b4) 🙏